go.mod

```diff
@@ -228,6 +228,8 @@ replace github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt
 
 replace github.com/portworx/sched-ops v0.0.0-20200831185134-3e8010dc7056 => github.com/portworx/sched-ops v0.20.4-openstorage-rc3 // required by rook v1.7
 
+replace github.com/IBM-Cloud/bluemix-go v0.0.0-20230601050310-eecebfbff63e => github.com/mihivagyok/bluemix-go v0.0.0-20230908132747-880981f4c25b
```

Comment:
I will remove this once the bluemix-go PR is merged.
https://github.com/IBM-Cloud/bluemix-go/pull/395